.github/workflows/doc.yml

```diff
@@ -97,6 +98,4 @@ jobs:
           overwrite: true
           retention-days: 7
```

Comment:
Should it also go down to 1 day?

Reply:
I figured the user documentation, since it's thinner than the dev (which contains perhaps hundreds of MB of include/use maps), can stay around longer. It's also more likely to be kept and uploaded with version releases.

Reply:
Ok.  It might help our future self to add some comments justify the choices.